utils/cp_parallel.py

```diff
@@ -98,6 +109,11 @@ def run_cellprofiler_parallel(
             "-i",
             path_to_images,
         ]
+
+        # if plugins_dir is provided, add the flag to find the plugins directory with given path
+        if plugins_dir:
+            command.extend(["--plugins-directory", plugins_dir])
+
```

Comment:
I would consider adding a raise error here if the directory does not exist so it is caught prior to entering into CellProfiler

Reply:
I think that this could also be solved by including the `.resolve(strict=True)` to the path. I think that is the simplest path forward.